src/utils/network/session-helper.js

```diff
@@ -216,7 +217,8 @@ export const setupAcl = (resourceWithAcl, webId, accessObject) => {
  * @param {URL} documentUrl - Url link to document container
  * @param {string} generateType - A string for "create" or "update"
  * @param {URL} webId - The webId of the user permissions are set to
- * @param {Access} accessObject - Access Object to use when creating the file
+ * @param {Access} accessObject - Access Object to set user permissions
+ * @param {Access} publicAccessObject - Access Object to set public permissions
```

Comment:
Allow ACL setup to add public access objects as well. It may be worth refactoring ACL management in a later PR so we have better utilities for handling user vs public vs organization permissions.

Reply:
Can use the same line for Line 221 for the updated JSDoc in setupAcl.